core/src/main/thrift/tabletingest.thrift

```diff
@@ -0,0 +1,130 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+namespace java org.apache.accumulo.core.tabletingest.thrift
+namespace cpp org.apache.accumulo.core.tabletingest.thrift
+
+include "data.thrift"
+include "security.thrift"
+include "client.thrift"
+include "manager.thrift"
+include "master.thrift"
+include "trace.thrift"
+include "tabletserver.thrift"
+
+exception ConstraintViolationException {
+  1:list<data.TConstraintViolationSummary> violationSummaries
+}
+
+enum TDurability {
+  DEFAULT = 0
+  SYNC = 1
+  FLUSH = 2
+  LOG = 3
+  NONE = 4
+}
+
+service TabletIngestClientService {
```

Comment:
It's a bit tedious to compare these directly with the previous code. It would help to know in advance: did you add/remove anything in here, or simply relocate them from the old consolidated service definitions to the new separate service definitions? I don't see a problem with the relocations in principle, so I'd like to focus on looking at added/removed/substantively changed things, if you can point me to them in advance.

Reply:
I just decomposed tabletserver.thrift into multiple services. I don't think anything should have been added or removed, just relocated.